packages/extension-browser/src/devtools/views/controls/print-link.css

```diff
@@ -0,0 +1,23 @@
+.root {
+    --border: 1px solid var(--link-color);
+
+    border: none;
+    border-bottom: var(--border);
```

Comment:
Pretty minor, but some platforms have some rounding on their buttons we'll need to disable here to avoid a slight distortion of the underline:
![image](https://user-images.githubusercontent.com/785009/92946718-a0aa5300-f41c-11ea-9bdd-ab15a66c6f8f.png)


```suggestion
    border-bottom: var(--border);
    border-radius: 0;
```